### ISSUE TITLE
Header Injection not implemented

### ISSUE THREAD

**Issue by krissik** [collective/collective.easyform#146]
I try to inject some css to a form.
![header_injection](https://user-images.githubusercontent.com/5301889/55794744-5229b400-5ac6-11e9-818e-cc4214bdc15e.jpg)

But it is not working. The injected css is not in the source of the page.

I am using python3, Plone 5.2rc1 (5202) and master branch of collective.easyform (currently https://github.com/collective/collective.easyform/commit/409648eeede495cd0216491fb892bfdcf7fe75a0 )

**Comment by krissik** [comment]
I will try to implement this - but I have no idea which template file is the correct one. Maybe someone can point me to it.

Related to that I do not understand 
```html
 <div id="content-core" tal:content="structure view/contents" />
``` 
at
https://github.com/plone/plone.app.z3cform/blob/master/plone/app/z3cform/templates/layout.pt:L13
`view` is `Products.Five.browser.metaconfigure.MyFormWrapper` object. But I can not find what  `contents` is / where it is implemented.

**Comment by krissik** [comment]
> But I can not find what `contents` is / where it is implemented.
Finally found it at:
https://github.com/zopefoundation/plone.z3cform/blob/master/src/plone/z3cform/layout.py#L57

**Comment by petschki** [comment]
closed by #152
